src/zkevm_specs/evm/execution/__init__.py

```diff
@@ -73,4 +74,5 @@
     ExecutionState.SHR: shr,
     ExecutionState.SDIV_SMOD: sdiv_smod,
     ExecutionState.STOP: stop,
+    ExecutionState.RETURN: return_,
```

Comment:
Just curious, why the suffix `_`?

Reply:
I just added a note explaining this in https://github.com/privacy-scaling-explorations/zkevm-specs/pull/229/commits/4e4114d22d2c7bfa55c6e3d71fe7ca1ea76c1c68

```
# NOTE: This python file is called `return_.py` and the opcode gadget is called
# `return_` because `return` is a reserved keyword in python and so can't be
# used in the code (but with an underscore suffix it's OK).
```